analyzer/codechecker_analyzer/cmd/checkers.py

```diff
@@ -256,7 +256,7 @@ def __get_detailed_checker_info(
         config_handler = analyzer_config_map.get(analyzer)
```

Comment:
I can't see if this can be factored out yet. This map contains the config handlers which we need in order to set the initially enabled checker list through `initialize_checkers()`. Unfortunately this function is still part of the config handler.